bentoml/_internal/bento/docker/templates/miniconda_alpine.j2

```diff
@@ -1,6 +1,6 @@
 {% extends "base_alpine.j2" %}
-{% set __python_version_txt__=expands_bento_path("env", "python", "version.txt") %}
-{% set __environment_yml__=expands_bento_path("env", "conda", "environment.yml") %}
+{% set __python_version_txt__=expands_bento_path("env", "python", "version.txt", bento_path=bento__path) %}
```

Comment:
can't we get this from env?  we still want to create the version.txt file but the dockerfile doesn't need to rely on that